pkg/access/commands/reset.go

```diff
@@ -20,17 +26,39 @@ func AccessReset() *cobra.Command {
 				return err
 			}
 
-			id := args[0]
-			_, _, err = client.ClientAPI.AccessSessionsAPI.ResetAccessSessionCredentials(cmd.Context(), id).Execute()
+			sessionID := args[0]
+
+			_, _, err = client.ClientAPI.AccessSessionsAPI.ResetAccessSessionCredentials(cmd.Context(), sessionID).Execute()
 			if err != nil {
 				return err
 			}
 
-			_, err = fmt.Fprintln(cmd.OutOrStdout(), "credentials reset request has been submitted")
+			_, err = fmt.Fprintln(cmd.OutOrStdout(), "credentials reset request has been submitted, waiting until finished...")
 			if err != nil {
 				return err
 			}
 
+			retries := 0
+			for {
+				session, _, err := client.ClientAPI.AccessSessionsAPI.GetAccessSession(cmd.Context(), sessionID).Execute()
+				if err != nil {
+					return fmt.Errorf("access session with id %s not found", sessionID)
+				}
+
+				if session.Credentials.IsSet() && session.Credentials.Get().Status == newCredentialsStatus {
+					break
+				}
+
+				time.Sleep(1 * time.Second)
+
+				retries++
+				if retries > maxWaitAttempts {
+					return fmt.Errorf("timeout while waiting for credentials to reset")
+				}
```

Comment:
I would check time and not retries
if there requests for sesver take long for some reason it will be stuck for long